Dockerfile.spdz

```diff
@@ -12,8 +12,8 @@ FROM ubuntu@sha256:10cbddb6cf8568f56584ccb6c866203e68ab8e621bb87038e254f6f27f955
 
 ARG VERSION
 ARG ARTIFACT_NAME="spdz"
-ARG SPDZ_VERSION="5350e66"
-ARG ARTIFACT_TAG="5350e66_no-offline"
+ARG SPDZ_VERSION="642d11f"
+ARG ARTIFACT_TAG="642d11f_no-offline"
```

Comment:
No offline is the new normal. Would it make sense to remove the "no-offline" qualifier?

Reply:
It definitely would, however we already have an image for the given version published. Reusing the tag would result in a conflict. I would therefore vote for keeping the qualifier and remove it once we update the spdz version itself

Reply:
Can you maybe add that plan as a comment or create an issue to track it?

Reply:
I have added a comment to follow up on this.